pysatModelUtils/tests/test_model_utils.py

```diff
@@ -1,18 +1,13 @@
-import numpy as np
-import sys
-
-from nose.tools import assert_raises, raises
-import pandas as pds
+from nose.tools import raises
 
 import pysat
-from pysat import model_utils as mu
+import pysatModelUtils as mu
 
 
 class TestBasics():
     def setup(self):
         """Runs before every method to create a clean testing setup."""
-        self.testInst = pysat.Instrument(platform='pysat',
-                                         name='testing',
+        self.testInst = pysat.Instrument('pysat', 'testing',
```

Comment:
as a matter of curiosity, why remove the platform, name keywords?

Reply:
This was back in my "try anything to fix" stage. Will fix.